src/huggingface_hub/constants.py

```diff
@@ -30,6 +30,7 @@
 
 
 HUGGINGFACE_CO_URL_TEMPLATE = ENDPOINT + "/{repo_id}/resolve/{revision}/{filename}"
+HUGGINGFACE_HEADER_X_REPO_COMMIT = "X-Repo-Commit"
```

Comment:
Could you add a comment specifying what that will be used for?

Reply:
actually let's just remove it from this PR and i'll add to another one